inference-engine/ie_bridges/java/org/intel/openvino/TensorDesc.java

```diff
@@ -0,0 +1,68 @@
+package org.intel.openvino;
+
+import java.util.concurrent.BlockingDeque;
+
+public class TensorDesc extends IEWrapper {
+
+    public TensorDesc(long addr){
+       super(addr);
+    }
+
+    public TensorDesc(Precision precision, int[] dims, Layout layout) {
+        super(GetTensorDesc(precision.getValue(), dims, layout.getValue()));
+    }
+
+    public int[] getDims() {
+        return GetDims(nativeObj);
+    }
+
+    public void setDims(int[] dims) {
+        setDims(nativeObj, dims);
+    }
+
+    public void setLayout(Layout layout) {
+        setLayout(nativeObj, layout.getValue());
+    }
+
+    public Layout getLayout(){
+        return Layout.valueOf(getLayout(nativeObj));
+    }
+
+    public void setPrecision(Precision precision) {
+        setPrecision(nativeObj, precision.getValue());
+    }
+
+    public Precision getPrecision(){
+        return Precision.valueOf(getPrecision(nativeObj));
+    }
+
+    public void reshape(int[] dims, Layout layout){
+        reshape(nativeObj, dims, layout.getValue());
+    }
+
+    public void reshape(int[] dims, BlockingDesc bDesc){
```

Comment:
@ilyachur @vinograd47 do we need `BlockingDesc` in java? This API looks advanced and I think we can skip it in Java

@likholat @dkurt is it used in samples / demo?

Reply:
This is a part of public API

Reply:
In general case use can use BlockingDesc in extensions but I am not sure that it is a case for Java API.

Reply:
@dkurt I understand that this is a part of C++ API, but in Java we can export only needed API which not over-complicate the Java bindings. So, I am trying to understand - is it used in Java? Or just because we have such method in C++

Reply:
I have not seen. @likholat, let's remove this method if it's not used.